sdk/storage/azure-storage-file-share/tests/test_file.py

```diff
@@ -1415,19 +1652,26 @@ def test_copy_file_async_private_file_with_sas(self, storage_account_name, stora
         copy_resp = file_client.start_copy_from_url(source_url)
 
         # Assert
-        self.assertTrue(copy_resp['copy_status'] in ['success', 'pending'])
+        assert copy_resp['copy_status'] in ['success', 'pending']
         self._wait_for_async_copy(self.share_name, target_file_name)
 
         actual_data = file_client.download_file().readall()
-        self.assertEqual(actual_data, data)
+        assert actual_data == data
 
     @FileSharePreparer()
-    def test_abort_copy_file(self, storage_account_name, storage_account_key, secondary_storage_account_name, secondary_storage_account_key):
+    @recorded_by_proxy
+    def test_abort_copy_file(self, **kwargs):
+        storage_account_name = kwargs.pop("storage_account_name")
+        storage_account_key = kwargs.pop("storage_account_key")
+        secondary_storage_account_name = kwargs.pop("secondary_storage_account_name")
+        secondary_storage_account_key = kwargs.pop("secondary_storage_account_key")
+
         self._setup(storage_account_name, storage_account_key, secondary_storage_account_name, secondary_storage_account_key)
-        data = b'12345678' * 1024 * 1024
```

Comment:
Hmm my guess is that this will be flaky in the pipeline with a smaller file size since it requires the copy to take enough time so it can be aborted. Maybe we should put this back to `* 1024` but mark it as Live only and delete the recording since we really shouldn't have the 8 MiB recording?

Reply:
Yeah I can see that. From my local testing, I found that 256 (so roughly 2MB recording) hasn't flaked on me and is a 75% reduction in recording size, so maybe we can leave it and see how it plays out? Then if it goes wrong, I'll go back and make it Live only (should be easy resolution since we know we are running this little "experiment")